README.md

```
@@ -50,6 +50,14 @@ cd packages/webapp && nix develop -c npm run dev
 nix develop .#tauri-shell --command cargo tauri dev
 ```
 
+### Optional: Direnv
+
+To use with [Direnv](https://direnv.net/), make sure you have it installed and run `direnv allow`.
+
+Benefits:
+- Automatically loads Nix dev shells, which allows you to run commands without having to prefix them with `nix develop`.
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)